src/resmom/linux/mom_mach.c

```diff
@@ -4568,17 +4568,18 @@ static const char *physmem(
     }
 
   mem_total = 0;
-
+  int test_file = 0;
 #ifdef NUMA_SUPPORT
 
   for (i = 0; i < node_boards[numa_index].num_nodes; i++)
 #endif /* NUMA_SUPPORT */
     {
 #ifdef NUMA_SUPPORT
-    if (!(fp = fopen(node_boards[numa_index].path_meminfo[i],"r")))
+    test_file = !(fp = fopen(node_boards[numa_index].path_meminfo[i],"r"));
```

Comment:
...wouldn't it be cleaner to put the #ifdef portion around *just* the part that changes?  Or set `path_meminfo to `node_boards[numa_index].path_meminfo[i]` inside an #ifdef and leave the `if` outside it?